source/usage-examples/updateMany.txt

```diff
@@ -0,0 +1,26 @@
+=====================
+Update Many Documents
+=====================
+
+.. default-domain:: mongodb
+
+You can update multiple documents using the ``updateMany()`` method. 
+``updateMany()`` accepts a filter object, and updates the first document
```

Comment:
I'd be willing to be that updateMany actually updates all documents that match the filter, not just the first

Reply:
You're right - silly mistake of me (probably copy & paste job). Fixed